src/utils/buffer-helper.ts

```diff
@@ -33,10 +42,21 @@ export class BufferHelper {
     return false;
   }
 
-  static bufferInfo (media, pos, maxHoleDuration) {
+  static bufferInfo (
+    media: Bufferable,
+    pos: number,
+    maxHoleDuration: number
+  ): {
+    len: number,
+    start: number,
+    end: number,
+    nextStart?: number,
+  } {
     try {
       if (media) {
-        let vbuffered = media.buffered, buffered = [], i;
+        let vbuffered = media.buffered;
+        let buffered: BufferTimeRange[] = [];
+        let i: number;
         for (i = 0; i < vbuffered.length; i++) {
```

Comment:
```suggestion
        for (let i = 0; i < vbuffered.length; i++) {
```
then yu don't need to define it above